lib/core/presentation/common_widgets/appBar.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+class MyAppBar extends AppBar {
+  String apptitle = 'Get The Job!';
+
+  MyAppBar({this.apptitle})
+      : super(
+          backgroundColor: Colors.teal,
+          title: Text(
+            'Get The Job!',
+            style: TextStyle(fontSize: 20),
+          ),
+        );
+}
```

Comment:
run "Reformat code with dartfmt", I guess that something is wrong with style

Reply:
My android studio forced this style when i saved file or run `flutter format` 😕